src/core/Icon/utils.ts

```diff
@@ -0,0 +1,61 @@
+import { ColorClass } from "../styles/colors/types";
+import { IconName } from "./types";
+
+export const defaultIconSecondaryColor = (name: IconName) => {
+  const colorLookup = {
+    "icon-gui-check-circled-fill": "text-white",
+  } as Record<string, ColorClass>;
+
+  return colorLookup[name];
+};
+
+// This is a mapping of older icon names to their newer HeroIcon counterparts.
+export const guiIconAliases: Record<string, string> = {
```

Comment:
We could quite possibly paste this as a JSON file in the website repo for use by `Icon` view component...

Reply:
Tbf the aliasing is a temporarily measure to assist with migration. The better thing to do is a big ol search and replace as a measure of upgrading to this package version